pkgs/top-level/release-cuda.nix

```diff
@@ -69,98 +72,147 @@ let
     ;
 
   # Package sets to evaluate whole
+  # Derivations from these package sets are selected based on the value
+  # of their meta.{hydraPlatforms,platforms,badPlatforms} attributes
   packageSets = builtins.filter (lib.strings.hasPrefix "cudaPackages") (builtins.attrNames pkgs);
-  evalPackageSet = pset: mapTestOn { ${pset} = packagePlatforms pkgs.${pset}; };
+  evalPackageSetPlatforms = lib.genAttrs packageSets (pset: packagePlatforms pkgs.${pset});
+
+  # Explicitly select additional packages to also evaluate
+  # The desired platforms must be set explicitly here
+  evalIndividualPackagePlatforms = {
+    blas = linux;
```

Comment:
There's probably a way to modify `packagePlatforms` or some other function that avoids indentation changes?

Reply:
The indentation change happens, because I want to apply the filtering function **after** the manually specified package set and the `cudaPackages.*`-recursed package set are merged, but **before** `mapTestOn` is applied.

I am not sure, what are you suggesting regarding `packagePlatforms`. That function is only applied to the `cudaPackages.*`-recursed package set, not to the manually-specified package set. I suppose, we could also use `packagePlatforms` for the manually-specified set, but this might be outside the scope of this PR.

Reply:
Did I answer your question? If so, please mark this as resolved.